lib/request_code.dart

```diff
@@ -25,7 +25,9 @@ class RequestCode {
     _code = null;
 
     final urlParams = _constructUrlParams();
-    final launchUri = Uri.parse('${_authorizationRequest.url}?$urlParams');
+    final urlCustomParams = _constructCustomUrlParams();
```

Comment:
This all looks fine for mobile, but can you also support this on flutter web?

Reply:
Would that involve changing the msalconfig.dart ? I could try but I have no experience with flutter web,